q2_vizard/plugin_setup.py

```diff
@@ -65,12 +66,56 @@
                      ' plotted on the x-axis.',
         'y_measure': 'Numeric measure from the input Metadata that should be'
                      ' plotted on the y-axis.',
-        'color_by_group': 'Categorical measure from the input Metadata that'
-                          ' should be used for color-coding the scatterplot.',
+        'color_by': 'Categorical measure from the input Metadata that'
+                    ' should be used for color-coding the scatterplot.',
         'title': 'The title of the scatterplot.'},
     name='2D Scatterplot',
     description='Basic 2D scatterplot for visualizing two numeric Metadata'
                 ' measures with optional categorical color grouping.',
     examples={'scatterplot_defaults': ex.scatterplot_defaults,
               'scatterplot_all_measures': ex.scatterplot_all_measures}
 )
+
+
+plugin.visualizers.register_function(
+    function=lineplot,
+    inputs={},
+    parameters={
+        'metadata': Metadata,
+        'x_measure': Str,
+        'y_measure': Str,
+        'replicate_method': Str % Choices('none', 'median', 'mean'),
+        'facet_by': Str,
+        'title': Str
+    },
+    parameter_descriptions={
+        'metadata': 'Any metadata-like input with at least two'
+                    ' numeric measures for visualizing.',
+        'x_measure': 'Numeric measure from the input Metadata that should be'
+                     ' plotted on the x-axis.',
+        'y_measure': 'Numeric measure from the input Metadata that should be'
+                     ' plotted on the y-axis.',
+        'replicate_method': 'The method for averaging replicates'
+                            ' if present in the chosen `x_measure`.'
+                            ' Available methods are `median` and `mean`.',
+        'facet_by': 'Categorical measure from the input Metadata that'
+                    ' should be used for faceting the lineplot.',
+        'title': 'The title of the lineplot.'},
+    name='Lineplot',
+    description='Basic lineplot for visualizing two numeric Metadata'
+                ' measures with optional faceting.',
+    examples={
+        'lineplot_median_replicates_with_faceting':
+            ex.lineplot_median_replicates_with_faceting,
+        'lineplot_mean_replicates_with_faceting':
+            ex.lineplot_mean_replicates_with_faceting,
+        'lineplot_median_replicates_no_faceting':
+            ex.lineplot_median_replicates_no_faceting,
+        'lineplot_mean_replicates_no_faceting':
+            ex.lineplot_mean_repliates_no_faceting,
+        'lineplot_no_replicates_with_faceting':
+            ex.lineplot_no_replicates_with_faceting,
+        'lineplot_no_replicates_no_faceting':
+            ex.lineplot_no_replicates_no_faceting
+    }
```

Comment:
Maybe `{f.__name__: f for f in [ex.lineplot_median_replicates_with_faceting, ...]}` would make life easier.